apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -50,6 +50,12 @@ const getFile = (receipt, contacts) => {
     const file = propertyContacts.length > 0
```

Comment:
We need to check if Contact is verified

Reply:
This is a part of query in main function:
const contacts = await Contact.getAll(context, {
                    phone: context.authedItem.phone,
                    isVerified: true,
                })